components/ILIAS/Authentication/templates/default/tpl.session_statistics_center.html

```diff
@@ -12,6 +12,4 @@ <h3>{TIMEFRAME}</h3>
 			<li>{CAPTION_CLOSED_DETAILS}: {VALUE_CLOSED_DETAILS}</li>
 			<!-- END closed_details -->
 		</ul>		
-		<p>{CAPTION_MAXED_OUT_COUNTER}: {VALUE_MAXED_OUT_COUNTER}</p>

```

Comment:
Maybe we should keep this, so the "Session Statistics" can work with the old values.

Reply:
I couldn't find any code line where this variable is still set.